airbyte-integrations/connectors/source-hubspot/source_hubspot/source.py

```diff
@@ -220,10 +220,49 @@ def streams(self, config: Mapping[str, Any]) -> List[Stream]:
 
         return available_streams
 
+    # set of standard stream names to check for custom object name collisions
+    STANDARD_STREAM_NAMES = {
```

Comment:
My only concern is that this is hardcoded. Although the streams won't change very frequently, it's easy to lose track of this dependency. I don't think the stream object has access to the catalog, which makes complicated to solve the problem. @airbytehq/critical-connectors  any suggestion?

Reply:
Agreed, I'd love a more future-proof solve. I didn't see an easy way to get at the basic stream list though.